GitUp/Application/Document.m

```diff
@@ -880,11 +886,11 @@ - (void)_showHelpWithIdentifier:(NSString*)identifier {
     XLOG_DEBUG_UNREACHABLE();
   }
   if (showHelp) {
-    NSRect contentBounds = [_mainWindow.contentView bounds];
```

Comment:
I had problems with `NSRect rect = view.bounds` that doesn't compile in Xcode 6 or 7, I don't remember, so you have to do do `[view bounds]` instead.